Add features to distill template

ShuhongDai opened the pull request:
While using the distill template, I noticed it lacked many features available in the regular post template, such as typograms, echarts, and other interactive elements. To address this, I **included the necessary scripts and dependencies for these features in** _distill.liquid_, referencing them from misc.liquid. Now, the distill template offers the same interactive functionality as the post template.

george-gca commented:
Have you tested this? Also, can you create maybe a new distill post with all these included in it, so we can demonstrate how to use and how they'll appear?

ShuhongDai commented:
1. After testing, I verified that currently only the scripts _{% include scripts/badges.liquid %}_ and _{% include scripts/pseudocode.liquid %}_ **cannot be used** in Distill. Therefore, I removed these two lines of code from _distill.liquid_.

2. The other scripts have been **validated successfully**. I updated _2018-12-22-distill.md_ and added a tutorial in the blog on how to use these scripts.

george-gca commented:
Just missing prettier complaints and it will be merged.

ShuhongDai commented:
I have reviewed the code and fixed the prettier issues.

ShuhongDai commented on package-lock.json:
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "main",
+  "name": "al-folio",
```
This change was an unintended modification caused by running `Prettier`. The `name` field in `package-lock.json` was updated unnecessarily, but this change doesn't serve any purpose. This was my oversight. 😳

george-gca replied:
Can you remove this?

ShuhongDai replied:
This part has been reverted.

ShuhongDai replied:
This part has been reverted.

george-gca commented:
Just missing the prettier complaints.

ShuhongDai commented:
> Just missing the prettier complaints.

Locally, Prettier confirms all files are correctly formatted, and the version I pushed passed Prettier checks on my machine. However, the GitHub Action still reports a Prettier failure.

<img width="504" alt="截屏2024-12-04 12 47 28" src="https://github.com/user-attachments/assets/d1f90af9-02de-43d3-9fee-fafc5255b015">

george-gca commented:
This error was already fixed in the main branch. Just rebase your branch with `main` and it should fix this.

george-gca commented:
Thanks for your contribution.
